tests/test.rb

```diff
@@ -2,8 +2,9 @@
 require 'safe_yaml'
 
 data = SafeYAML.load_file '_data/project_filter.yml', safe: true
-expect_size = data.size
-projectdir = Dir.new('_site/dashboard/project')
+redirects = Dir.open('pages/project').count-2
```

Comment:
This `count-2` seems a bit fragile. Perhaps we shouldn't have individual `.md` files that contain the `redirect_to:` front matter, but we can use a plugin to generate `Jekyll::Page` objects from a `_data/redirect_pages.yml` or some such. Then we can also replace this `count-2` with the number of objects loaded from that data file.